media-proxy/src/api_server_tcp.cc

```diff
@@ -134,63 +136,9 @@ void* msg_loop(void* ptr)
                 if (it->id == session_id) {
```

Comment:
please change `Failed to start MTL session.` to `Failed to start a session.`  a few lines above. ^^^^ 
Thanks. :)

Reply:
done